via/get_url_details.py

```diff
@@ -17,6 +17,12 @@
     r"^https://drive.google.com/uc\?id=(.*)&export=download$", re.IGNORECASE
 )
 
+# The Chrome user-agent as of 24/06/2020
+BACKUP_USER_AGENT = (
+    "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) "
+    "snap Chromium/83.0.4103.106 Chrome/83.0.4103.106 Safari/537.36"
+)
```

Comment:
I don't think you can make a request without sending a User-Agent unless you are using curl or something, but better safe than sorry.